tests/sentry/integrations/slack/test_utils.py

```diff
@@ -103,3 +105,124 @@ def test_rate_limiting(self):
         )
         with pytest.raises(ApiRateLimitedError):
             get_channel_id(self.organization, self.integration, "@user")
+
+
+class GetChannelIdFasterTest(GetChannelIdTest):
+    @with_feature("organizations:slack-use-new-lookup")
```

Comment:
You don't need the feature decorator on the setup and tear down methods, or actually even any of the tests that don't directly call `get_channel_id`.